thrift/atoms/tenfour.thrift

```diff
@@ -0,0 +1,61 @@
+namespace * tenfour
+
+include "../shared.thrift"
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+struct Content {
```

Comment:
I don't think naming `Content` is a good idea, as this already use all over the place in both `flexible` and  `content-api`